06-building-a-program-start-to-finish/writing-tests/output/invertedIndexTests.js

```diff
@@ -43,7 +43,7 @@ describe('Tests the invertedIndex.js class: ', () => {
     });
   });
 
-  describe('Checks if value returned by getIndexedWords() is an object', () => {
+  describe('Checks if value returned by getIndexedWords()', () => {
     const returnedObject = invertedIndexTrue.getIndexedWords();
```

Comment:
Almost there with the name. You can ask your peers for the convention for naming test file.